std/log/handlers.ts

```diff
@@ -179,6 +179,7 @@ export class RotatingFileHandler extends FileHandler {
         }
       }
     }
+    this.#currentFileSize = (await stat(this._filename)).size;
```

Comment:
As a further efficiency, `#currentFileSize` could be initialised to 0, and then you only need to call the `await stat(this._filename)` if the mode is 'a' (append), making the code something like this:
```
#currentFileSize = 0;
...
if (this._mode === "w") {
...  // New file will be created, no need to look up file size
} else if (this._mode === "x") {
...  // New file will be created, no need to look up file size
} else {
   // Possibly appending to existing populated file, need to get size
   this.#currentFileSize = (await stat(this._filename)).size;
}
```

Reply:
Thanks for help